.pipelines/templates/template-deploy-int-env.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+  domainName: ""
+  rgPrefix: ""
+  workingDirectory: ""
+  location: ""
+  mdmFrontendUrl: ""
+  rpImageACR: ""
+  rpImagePullJSONSPN: ""
+  azureFpClientId: ""
+  azureDevOpsJSONSPN: ""
+  mdsdMonitoringEnviroment: ""
+  mdsdConfigVersion: ""
+  rpMode: ""
+  sshPublicKey: ""
+  pullSecret: ""
+  subscriptionId: ""
+steps:
+  - script: |
+      set -eu
+      cd ${{ parameters.workingDirectory }}
+      
+      COMMIT=$(git rev-parse --short HEAD)$([[ $(git status --porcelain) = "" ]] || echo -dirty)
+
+      echo ${{ parameters.rpImagePullJSONSPN }} | base64 -d -w 0 > pull-spn.json
+      pull_spn_client_id=$(cat pull-spn.json | jq -r '.clientId')
+      pull_spn_client_secret=$(cat pull-spn.json | jq -r '.clientSecret')
+      export RP_IMAGE_AUTH=$(echo "$pull_spn_client_id:$pull_spn_client_secret" | base64 -w 0)
+      export PULL_SECRET=$(echo ${{ parameters.pullSecret }} | sed 's/"/\\\\\\"/g')
```

Comment:
yikes - why is the sed needed?

Reply:
It isn't working yet. But the reason is this is a json value. And I'm trying to pass it through json so I need to escape it.  When I passed it from command line in python I could just escape it once.  I haven't been able to preserve the "s yet going through the json parameter file.  How do you guys set it deploying to prod?

Reply:
got it working. Ended up using base64 encoded json formatted how I wanted in keyvault. so no more sed